src/content/reference/cluster-upgrades/_index.md

```diff
@@ -107,7 +107,7 @@ In particular this means:
 - As a consequence of draining, Pods running on a node will be rescheduled to other nodes.
 - Once the master node is taken down and recreated, the Kubernetes API will be unavailable for a short time.
 
-**Note**: Currently tenant clusters have one master node each. We have plans on our roadmap to allow for multiple master nodes, keeping the Kubernetes API accessible during an upgrade and increasing the resilience in case of a machine failure.
+**Note**: By default, tenant clusters have one master node each. Consider using High Availability masters setup to avoid API downtime during upgrades. (See: [High availability Kubernetes masters](https://docs.giantswarm.io/basics/ha-masters/))
```

Comment:
@paurosello WDYT?

Reply:
looks better!

Reply:
Thanks for spotting this! I missed that in my last edit.

You can even direct readers to `#checklist-ha-masters` here on the same page, not switching context too much. No need for parantheses in my opinion.

Reply:
@marians can you make the change please? :)